include/option/option_boolean.h.in

```diff
@@ -5,7 +5,7 @@
 #ifdef __cplusplus
 
 namespace option {
-    constexpr bool ${OPTION_NAME_lowered} = ${OPTION_ENABLED};
+    inline constexpr bool ${OPTION_NAME_lowered} = ${OPTION_ENABLED};
```

Comment:
Does it really improve the generated code? Or was the intent different at this spot?

Reply:
While I have not checked generated code difference in this case, it's a good idea to inline global constexpr variables, per the article:
https://quuxplusone.github.io/blog/2022/07/08/inline-constexpr/
A tl:dr of it could be that:
- inline constexpr variables result in potentially smaller code, best observed with big variables that in case of no inline are duplicated in each TUs (and therefore increasing binary size). Linker de duplicates inline variables leaving only one. 
- (cosmetic) ccls stops complaining that the variable is unused

Reply:
Thanks :+1: